### PR TITLE
issue=#452 doc.doc-readme

### PR DESCRIPTION
**Pull request by taocp**
#452

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -1,52 +1,53 @@
...
-## TodoList
-1. Master的设计与实现（职责、功能、每个功能怎么实现的、为什么这么实现、还有哪些遗留问题）
-1. Tabletserver设计与实现
-1. SDK的结构化文档（可折叠，可搜索，类是windows的帮助）
-1. 性能数据，不同场景下的测试数，瓶颈分析
-1. 性能方面设计折衷（怎样实现的高性能随机读写）
-1. 监控系统的安装和使用
-1. 快速、低成本Split&Merge的实现原理
+1. SDK manual(user friendly: e.g. searchable, foldable)
+
```
Comment (00k) [inline doc/README.md]: remove empty lines to make it  concise

File: doc/README.md [hunk 2]
```diff
@@ -1,52 +1,53 @@
...
+1. Performance benchmark on various enviornment
```
Comment (00k) [inline doc/README.md]: enviornment

File: doc/README.md [hunk 3]
```diff
@@ -1,52 +1,53 @@
...
+
+1. Trade-off about performance(how to implement high perfromance random read/write)
```
Comment (00k) [inline doc/README.md]: Design trade-off

File: doc/README.md [hunk 4]
```diff
@@ -1,52 +1,53 @@
-# Tera文档专区
+# Tera Docs
 
-## 简介
+## Introduction
 
-[系统设计](tera_design.md)
+[tera design](tera_design.md)
```
Comment (00k) [inline doc/README.md]: capitalize the first letter of every words between []
except for on/to/of/...

File: doc/README.md [hunk 5]
```diff
@@ -1,52 +1,53 @@
...
+## Performance Profiling
 
-## 性能优化
+[Read/Write](perf_opz.md)
 
-[针对表格不同读写特性的性能优化建议](perf_opz.md)
+## Inside Tera
 
-## 功能实现
+[how load balance works](load-balance.md)
```
Comment (00k) [inline doc/README.md]: "how load balance works" sounds like to explain why the load balance works (which is challenged by someone), not to explain how it implemented

File: doc/README.md [hunk 6]
```diff
@@ -1,52 +1,46 @@
...
+[Interaction Between Master, Tablet Server and Client](master-ts-client-interactive.md)
 
-[tablet的可用性统计实现](tablet-availability.md)
+## Version Management
 
-## 版本发布
+[Release Management](release_management.md)
 
-[版本发布及管理](release_management.md)
+## Todo List
```
Comment (caijieming-baidu) [inline doc/README.md]: 是否要改成待完善的文档，否则会有人质疑我们功能未实现

**Pull request merged**